docs/api/apiv3/paths/work_package_file_link_download.yml

```diff
@@ -0,0 +1,62 @@
+# /api/v3/work_packages/{id}/file_links/{file_link_id}/download
+---
+post:
+  summary: Creates a download uri of the linked file.
+  operationId: Work_Package_File_Link_Download
+  tags:
+    - Work Packages
+    - File links
+  description: |-
+    Creates a download uri of the origin file linked by the given file link. This uri depends on the storage type and
+    always is located on the origin storage itself.
```

Comment:
I'd say, the URI points always to the storage linked inside the file link, as the whole concept is built around the idea, of just saving a reference to the original file.

As for the idea, of instead of making a response containing the URI where to find that file, or directly returning a redirect, I'm just not sure, what we want as a behavior of our API.

Reply:
reworked static resource endpoints to return redirects